ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/PeriodicRecovery.java

```diff
@@ -2,6 +2,7 @@
    Copyright The Narayana Authors
    SPDX-License-Identifier: Apache-2.0
  */
+
```

Comment:
A little bit of formatting that could be reverted, please?

Reply:
(and on line 36 of this file)

Reply:
Done, thanks Tom